pyproject.toml

```diff
@@ -3,21 +3,25 @@ name = "synchronicity"
 version = "0.8.3"
 description = "Export blocking and async library versions from a single async implementation"
 readme = "README.md"
+authors = [
+    { name = "Modal Labs" }
+]
+requires-python = ">=3.8"
+dependencies = [
+    "sigtools>=4.0.1",
+    "typing-extensions>=4.12.2",
+]
 classifiers = [
   "Operating System :: OS Independent",
   "License :: OSI Approved :: Apache Software License",
   "Programming Language :: Python :: 3",
 ]
 
-dependencies = ["sigtools==4.0.1", "typing_extensions>=4.6"]
-requires-python = ">=3.8"
 
 [build-system]
-requires = ["setuptools", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
```

Comment:
Look at this lil guy 
![image](https://github.com/user-attachments/assets/f8843084-f452-4692-9b39-8434d61a1454)